Misc/NEWS.d/next/Core and Builtins/2017-10-23-23-39-26.bpo-28936.C288Jh.rst

```diff
@@ -0,0 +1,2 @@
+Ensure that lexically first syntax error involving ``global`` or
+``nonlocal`` is detected first at a given scope. Patch by Ivan Levkivskyi.
```

Comment:
Mention that this is specific to parameters? Other errors already are detected first.